mantidproject/mantid
# Add ROI Detector ID field to Experiment lookup table

**Description of work.**
This PR adds the ROI Detector ID column to the Experiment Lookup table. It also passes this property through to the relevant algorithm when processing is performed.

**To test:**
1. Open the `ISIS Reflectometry` GUI
2. Type `INTER45455_inst` in the runs table and an angle of 0.5
3. Go to the experiment settings and create a setting for angle 0.5, and change the ROI Detector IDs column to `2001`
4. Go back to the runs table and process all
5. Open the history of the output workspace and check that `2001` has been passed into the algorithm

Part of #34110





---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
